backend/app/src/main/java/com/moh/phlat/backend/esb/json/OFRelationshipResponse.java

```diff
@@ -51,9 +52,10 @@ public void plrJsonToProcessData(String json) {
 			objectMapper.disable(DeserializationFeature.ADJUST_DATES_TO_CONTEXT_TIME_ZONE);
 			objectMapper.setSerializationInclusion(Include.NON_NULL);
 			
+			boolean hasError = false;
 			JsonNode root = objectMapper.readTree(json);
```

Comment:
let's initialize object mapper once in static block. Suggest moving the initialization to a single place where all the three classes can use. Either use inheritance or utility class.